src/main/java/io/confluent/kafkarest/ConsumerSubscriptionState.java

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright 2015 Confluent Inc.
```

Comment:
Please use `2016` instead of `2015`. The same is true for the other licenses agreements below. Thanks!